### DISCUSSION TITLE
EMP目前支援angular嗎?

### DISCUSSION THREAD

**Discussion by evolution9502** [empjs/emp#176] [Q&A]
目前看到支援的項目內似乎不包含angular
官方文檔裡也都沒提到是否支援angular
請問下技術上能實現嗎?

**Comment by ckken** [comment]
可以参考下 其他框架的插件 https://github.com/efoxTeam/emp/blob/main/packages/emp-frameworks/vue2/index.js

**Reply by kiddliu** [reply]
个人感觉不打开zone支持的Angular应该是可以的，不过这样使用Angular的意义也就不大了

**Reply by ckken** [reply]
可以考虑用mf 目前ng是支持接入的
